README.md

```diff
@@ -1,15 +1,25 @@
-PKCS11Key
-========
+# PKCS11Key
 
 The pkcs11key package implements a crypto.Signer interface for a PKCS#11 private key.
 
 If you are using Go modules, you should import this with the module-compatible
-path github.com/letsencrypt/pkcs11key/v4
+path `github.com/letsencrypt/pkcs11key/v4`.
 
-[![Build Status](https://travis-ci.org/letsencrypt/pkcs11key.svg)](https://travis-ci.org/letsencrypt/pkcs11key)
+# Testing
 
-## License summary
+* You will need to install SoftHSMv2.
+* Run `./test.sh`
+* If you need to regenerate key material, run the following and check the new files back into git.
+```
+cd v4/testdata
+openssl ecparam -name prime256v1 -genkey | openssl pkcs8 -topk8 -nocrypt > entropic_ecdsa.key
+openssl req -new -x509 -key entropic_ecdsa.key -out entropic_ecdsa.pem -days 1000 -subj /CN=entropic\ ECDSA
+openssl req -new -newkey rsa:2048 -nodes -x509 -keyout silly_signer.key -out silly_signer.pem -days 1000 -subj /CN=silly\ signer
+cd -
```

Comment:
(Not for this PR, but for the future: it might be cool to replace this comment and the comment in //v4/testdata/readme.md with a //v4/testdata/generate.go so that the instructions can be simplified to "if you need to regenerate key material, run go generate".)